pg_shard.c

```diff
@@ -2019,3 +2026,79 @@ PgShardProcessUtility(Node *parsetree, const char *queryString,
 								params, dest, completionTag);
 	}
 }
+
+
+/*
+ * PgShardProcessUtilityDropCommands is a helper function for PgShardProcessUtility.
+ * This function handles DROP utility commands. Currently, only handles DROP EXTENSION
+ * command.
+ */
+static void
+PgShardProcessUtilityDropCommands(Node *parsetree)
+{
+	DropStmt *dropStatement = (DropStmt *) parsetree;
```

Comment:
I think the idiom within PostgreSQL is to cast this _before_ calling the function so the function signature can be more specific.